internal/services/v1/relationships.go

```diff
@@ -353,8 +353,13 @@ func (ps *permissionServer) WriteRelationships(ctx context.Context, req *v1.Writ
 		writeUpdateCounter.WithLabelValues(v1.RelationshipUpdate_Operation_name[int32(kind)]).Observe(float64(count))
 	}
 
+	zedToken, err := zedtoken.NewFromRevision(ctx, revision, ds)
+	if err != nil {
+		return nil, ps.rewriteError(ctx, err)
+	}
+
 	return &v1.WriteRelationshipsResponse{
-		WrittenAt: zedtoken.MustNewFromRevision(revision),
```

Comment:
hmm, Im realizing we used `MustNewFromRevision` everywhere, when it's something that would panic in case of malformed proto encoding. I guess it worked because the middleware prevented any incorrect revision from making it through.

Reply:
Yeah, we could move away from the `Must` approach, but it shouldn't fail in "normal" operation